### PR TITLE
Add docs and tests for latest changes which adds current section

### PR DESCRIPTION
**Pull request by bharathappali**
The latest changes to recommendation API includes a `current` section, this PR adds remaining tests and Monitoring doc MD changes for the respective new introductions in the Recommendation API

### CODE REVIEW

File: tests/scripts/remote_monitoring_tests/rest_apis/test_list_recommendations.py [hunk 1]
```diff
@@ -973,6 +986,10 @@ def test_list_recommendations_with_changes_in_update_results(cluster_type: str):
 
             short_term_recommendation = data_section[str(end_time)]["duration_based"]["short_term"]
 
+            assert "variation" in short_term_recommendation
```
Comment (chandrams) [inline tests/scripts/remote_monitoring_tests/rest_apis/test_list_recommendations.py:989]: @bharathappali - As discussed, variations validation is common code and should be included in utils.py in validate_reco_json.  And value for amount needs to be validated in variation as below:

variation = recommended - current (if requests/limits is set)
variation = recommended (if requests/limits is not set)

**Comment by chandrams** [comment]
@bharathappali - Have you tested your test changes? I still see this error

    assert INFO_DURATION_BASED_RECOMMENDATIONS_AVAILABLE_CODE in high_level_notifications
E               AssertionError: assert '112101' in {'120001': {'code': 120001, 'message': 'There is not enough data available to generate a recommendation.', 'type': 'info'}}

File: tests/scripts/remote_monitoring_tests/rest_apis/test_list_recommendations.py [hunk 2]
```diff
@@ -934,6 +931,22 @@ def test_list_recommendations_with_changes_in_update_results(cluster_type: str):
                     memory_limit_entry = container_metrics[MEMORY_LIMIT_INDEX]
                     aggre_info = memory_limit_entry["results"]["aggregation_info"]
                     aggre_info["format"] = None
+            elif j == 109:
```
Comment (chandrams) [inline tests/scripts/remote_monitoring_tests/rest_apis/test_list_recommendations.py:934]: Please rename the function to test_list_recommendations_notification_codes or something meaningful

**Comment by bharathappali** [comment]
> @bharathappali - Have you tested your test changes? I still see this error
> 
> ```
> assert INFO_DURATION_BASED_RECOMMENDATIONS_AVAILABLE_CODE in high_level_notifications
> ```
> 
> E AssertionError: assert '112101' in {'120001': {'code': 120001, 'message': 'There is not enough data available to generate a recommendation.', 'type': 'info'}}

Thanks for the review @chandrams , The issue is now fixed with the latest commit pushed to the PR. Please check and lemme know if you still facing that issue.

File: tests/scripts/remote_monitoring_tests/rest_apis/test_list_recommendations.py (hunk 1 above)
Comment (bharathappali) [reply tests/scripts/remote_monitoring_tests/rest_apis/test_list_recommendations.py]: Added the generic check in the utils

File: tests/scripts/remote_monitoring_tests/rest_apis/test_list_recommendations.py (hunk 2 above)
Comment (bharathappali) [reply tests/scripts/remote_monitoring_tests/rest_apis/test_list_recommendations.py:934]: renamed it to `test_list_recommendations_notification_codes`. Thanks!

**Comment by bharathappali** [comment]
There was an issue with tests, that the tests are updating results for a container and validating results of another container. Thanks @chandrams for finding out the bug in the tests. I have add the fix suggested by @chandrams in the latest commit.

**Review by dinogun — APPROVED** [review]
LGTM

**Pull request merged**